src/Cache.ts

```diff
@@ -7,9 +7,11 @@ import * as crypto from 'crypto';
 
 const d = debug('@electron/get:cache');
 
-const defaultCacheRoot = envPaths('electron', {
-  suffix: '',
-}).cache;
+const defaultCacheRoot =
+  process.env.ELECTRON_GET_CACHE_DIR ||
```

Comment:
Do we need to be able to specify this via npm config like several other configuration options?

https://github.com/electron/get/blob/519d3cc56ae5e553a4952f5f687f094599cf473a/src/artifact-utils.ts#L35-L42

Reply:
I'd be happy for a second opinion on this because I'm not so familiar with this codebase, but -- at first read, I don't think so? It looks to me like line 39 in that snippet will cover any environment variable beginning with `ELECTRON_`, including this one.

Reply:
Well, we'd have to use that method in this callsite if we want to use functionality I linked. Unless I'm misunderstanding you.

Reply:
It's more likely that I'm the one misunderstanding, as I've not read `artifact-utils.ts` before.

**If** I'm reading this file correctly, it looks like mirrorVar() is used to build the URL path for a remote artifact and so a per-client destination directory setting would not need to be mapped. The server is just serving files; it doesn't need to know or care where the client stores them. There's no reason for its URLs' path to have any mapping to a per-client setting...?